src/Components/content.js

```diff
@@ -1,13 +1,40 @@
 import React, { Component } from 'react';
 import Source from './source';
+import './content.css';
 
 class Content extends Component {
   render() {
```

Comment:
Please move the "display logic" (toggle = true? then... : else ...) before the return statement and then pass the result of this logic (empty or xxx-toggle) to the div classNames properties.